### PR TITLE
Setup for website

### PR DESCRIPTION
**Pull request by A5rocks**
Opening this mainly for CI atm, but feel free to look over

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+The README has yet to be written.
```
Comment (BautistaDalmasso) [inline README.md]: We could leave this comment on the readme out I believe. It doesn't relate to setting up the website and I doubt we will forget to rewrite the README when the time comes.

File: server/main.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+SCRIPT = Path(__file__)
+PUBLIC_DIR = (SCRIPT / "../../public").resolve()
+
+app = FastAPI()
+
+app.mount("/static", StaticFiles(directory=PUBLIC_DIR, html=True))
+
+
+# small little redirect
```
Comment (AbooMinister25) [inline server/main.py:15]: I'm just nitpicking, but this comment isn't really neccessary when you have the docstring

**Review by AbooMinister25 — COMMENTED** [review]
For the most part everything looks good

**Review by AbooMinister25 — APPROVED** [review]
I'm going to go ahead and approve this, since content-wise everything looks good

**Pull request merged**